sw/host/opentitansession/src/main.rs

```diff
@@ -163,6 +162,11 @@ fn session_child(listen_port: Option<u16>, backend_opts: &backend::BackendOpts)
     // through the process of making this process a daemon, disconnected from the
     // terminal that was used to start it.
 
+    // All configuration files have been processed (relative to current direction), we can now
+    // drop the reference to the file system, (in case the admin wants to unmount while this
+    // daemon is still running.)
+    env::set_current_dir(&Path::new("/"))?;
```

Comment:
Nit: Since this actually takes `impl AsRef<Path>`, you could also pass in just `"/"`. Most things that take a Path are like this; similarly, the & is unnecessary; `Path:;new()` actually looks like
```rust
pub fn new<S: AsRef<OsStr> + ?Sized>(s: &S) -> &Path;
```

Reply:
Thanks, done.